docs/alerts.md

```diff
@@ -150,17 +158,17 @@ Elasticsearch will attempt to relocate shards away from a node for [reaching the
     - To allocate shards to this particular node, free up some spaces:
         - Increase disk space on all nodes.
         - If increasing disk space isn’t possible, then add a new data node to the cluster.
-        - If adding a new data node is problematic, then reduce the number of replicas to 1 if it is higher than 1.
-            - Preferably on older indices.
-            - Get the index settings:
+        - If adding a new data node is problematic, then decrease the total cluster redundancy policy
+            - Check current redundancyPolicy
               ```
-              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty
+              oc edit Elasticsearch -n openshift-logging
```

Comment:
we're missing the object name here:
`oc edit es elasticsearch -n openshift-logging`